packages/plugins/apollo-managed-federation/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@graphql-yoga/apollo-managed-federation",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```You don't need to bump the version here. When we release it, changeset will already bump it.
If you set it now, and we merge it, changeset will release it directly instead of creating a PR because changeset releases the version if it doesn't exist on NPM and there is no changeset at that moment under .changeset.